examples/CIB/ex5/input3d.one_body

```diff
@@ -0,0 +1,484 @@
+// ---------------------------------------------------------------------------
+
+// random-number generator parameters
+SEED = 0                                // RNG seed (0 for clock)
+
+// ---------------------------------------------------------------------------
```

Comment:
```suggestion
```
(we do not use this kind of formatting in the input files; lets keep things uniform by not having sections delineated this way)